Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -1392,6 +1392,60 @@ void SoftwareGLContext::gl_get_floatv(GLenum pname, GLfloat* params)
     }
 }
 
+void SoftwareGLContext::gl_get_booleanv(GLenum pname, GLboolean* data)
+{
+    RETURN_WITH_ERROR_IF(m_in_draw_state, GL_INVALID_OPERATION);
+
+    switch (pname) {
+    case GL_BLEND:
+        if (m_blend_enabled)
+            *data = GL_TRUE;
+        else
+            *data = GL_FALSE;
```

Comment:
```suggestion
        *data = m_blend_enabled ? GL_TRUE : GL_FALSE;
```

Really just a suggestion and personal taste, but this function is going to get pretty long eventually. Maybe take a few shortcuts?

Reply:
Good idea, bit of a habit from $job now....